src/simplay-web/src/event/ContainerSetCapacityEvent.ts

```diff
@@ -12,6 +17,14 @@ export class ContainerSetCapacityEvent extends Event {
     super(forId, timestamp, EventAction.CONTAINER_SET_CAPACITY, args);
   }
   execute(context: SimplayContext) {
-    throw new Error('Method not implemented.');
+    const entityDisplayObject = getEntityDisplayObjectById(
+      context,
+      this.forId
+    ) as ExtendedDisplayEntity;
+    const entity = context.simulationData.entities.find(
```

Comment:
This is a expensive, especially if we have a lot of entities, should we change simulationData.entities to a map? if you also think so lets create a task

Reply:
I agree. This is something that can be done once, when converting from SimulationDataSerialized to SimulationData during initialization of the SimulationSpooler. I'll create a task for this.